Source/Krypton Components/Krypton.Toolkit/Designers/Designers/KryptonStubDesigner.cs

```diff
@@ -0,0 +1,19 @@
+﻿#region BSD License
+/*
+ * 
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2024 - 2024. All rights reserved. 
```

Comment:
Please add "giduac" into this.. You did it ;-)

Reply:
> Please add "giduac" into this.. You did it ;-)

There you go.

Reply:
Might need to update the license going forward...